src/widgets/settingspages/NotificationPage.cpp

```diff
@@ -51,6 +51,14 @@ NotificationPage::NotificationPage()
                     "Show notification", getSettings()->notificationToast));
 #endif
 #ifdef Q_OS_WIN
+                settings.append(this->createCheckBox(
+                    "Create start menu shortcut (requires "
+                    "restart)",
+                    getSettings()->createShortcutForToasts,
+                    "When enabled, a shortcut will be created inside your "
+                    "start menu folder if needed by live notifications."
+                    "\n(On portable mode, this is disabled by "
+                    "default)"));
```

Comment:
You can use `SettingWidget` here, which means you wouldn't have to update the `createCheckBox` implementation - see https://github.com/Chatterino/chatterino2/blob/7fee3f7e375a099b5207c215389343f8f1d95bf2/src/widgets/settingspages/GeneralPage.cpp#L1231-L1235 as a reference

Reply:
nvm all good, you can't do that here yet